src/Core/src/Platform/Android/ButtonExtensions.cs

```diff
@@ -87,5 +88,13 @@ internal static void UpdateButtonBackground(this MaterialButton platformView, IB
 					platformView.BackgroundTintList = null;
 				});
 		}
+
+		internal static void UpdateRippleColor(this MaterialButton platformView, Color rippleColor)
```

Comment:
This is just when the ripple color updates, but what happens when you update the background after? I think the code above will overwrite as it uses the `platformView.RippleColor` to get the color. We can/should probably overwrite that or even use that property instead.